### ISSUE TITLE
align_multiline_parameters produces new line for promoted properties without any types.

### ISSUE THREAD

**Issue by nizacegodk** [erickskrauch/php-cs-fixer-custom-fixers#8]
When using `'ErickSkrauch/align_multiline_parameters' => ['variables' => true]` we get an unexpected style fixing for classes with promoted properties without a type: 

## Before:
```php
/**
 * @template T
 */
class Diff
{
    /**
     * @param T $was
     * @param T $expected
     */
    public function __construct(
        public $was,
        public $expected,
    ) {
    }
}
```

## After:
```php
/**
 * @template T
 */
class Diff
{
    /**
     * @param T $was
     * @param T $expected
     */
    public function __construct(
        public
        $was,
        public
        $expected,
    ) {
    }
}
```

## Expected (No change):
```php
/**
 * @template T
 */
class Diff
{
    /**
     * @param T $was
     * @param T $expected
     */
    public function __construct(
        public $was,
        public $expected,
    ) {
    }
}
```

**Comment by erickskrauch** [comment]
For some reason, I thought property promotion was impossible without specifying a type. But I double-checked and no, the type is optional. There is definitely an error here. I will fix it soon.

**Comment by erickskrauch** [comment]
@nizacegodk, released in 1.2.3. Thank you for the issue!
